spec/features/sessions_spec.rb

```diff
@@ -13,13 +12,16 @@
     )
   end
 
+  # this is needed to bypass the Setup steps
+  before do
+    create(:configuration, name: 'admin:password', value: ::BCrypt::Password.create('rspec_pass'))
```

Comment:
```suggestion
    create(:configuration, name: 'admin:password', value: ::BCrypt::Password.create(password))
```

Reply:
I had this originally, but it breaks the "bad password" spec, as the `password` variable gets updated inside the block, and you end up using the "bad" one to save the config.